hyperspy/_signals/signal1d.py

```diff
@@ -1532,6 +1533,63 @@ def estimating_function(spectrum,
 
     estimate_peak_width.__doc__ %= (SHOW_PROGRESSBAR_ARG, NUM_WORKERS_ARG)
 
+    def get_intensity(self,
```

Comment:
There is a test failure related to this function. As this is not how it is expected to work, I have left it.